### PR TITLE
Related on cd

### PR DESCRIPTION
**Pull request by Kalyani790**

### CODE REVIEW

File: app/components/shared/RelatedVideoArticles.tsx [hunk 1]
```diff
@@ -119,7 +119,8 @@ const RelatedVideoArticles = (props: RelatedVideoArticlesProps): any => {
         backgroundColor: backgroundColor,
         detailData: item,
         listCategoryArray: listCategoryArray ? listCategoryArray : [],
-        currentSelectedChildId: currentSelectedChildId ? currentSelectedChildId : 0
+        currentSelectedChildId: currentSelectedChildId ? currentSelectedChildId : 0,
+        fromCd:fromScreen == "ChildDevelopment" ? true :false
```
Comment (Akhror) [inline app/components/shared/RelatedVideoArticles.tsx:123]: Let's please add spaces:
fromCd: fromScreen == "ChildDevelopment" ? true : false

File: app/components/shared/RelatedArticles.tsx [hunk 2]
```diff
@@ -111,7 +111,8 @@ const RelatedArticles = (props: RelatedArticlesProps): any => {
         backgroundColor: backgroundColor,
         detailData: item,
         listCategoryArray: listCategoryArray ? listCategoryArray : [],
-        currentSelectedChildId: currentSelectedChildId ? currentSelectedChildId : 0
+        currentSelectedChildId: currentSelectedChildId ? currentSelectedChildId : 0,
+        fromCd:fromScreen == "ChildDevelopment" ? true :false
```
Comment (Akhror) [inline app/components/shared/RelatedArticles.tsx:115]: fromCd: fromScreen == "ChildDevelopment" ? true : false

File: app/screens/home/DetailsScreen.tsx [hunk 3]
```diff
@@ -63,8 +63,8 @@ export type RelatedArticlesProps = {
   currentSelectedChildId?: any;
 }
 const DetailsScreen = ({route, navigation}: any): any => {
-  const {headerColor, fromScreen, backgroundColor,detailData, listCategoryArray, selectedChildActivitiesData, currentSelectedChildId} = route.params;
-  console.log(detailData,"..detailData...",fromScreen,"...fromScreen..");
+  const {fromCd,headerColor, fromScreen, backgroundColor,detailData, listCategoryArray, selectedChildActivitiesData, currentSelectedChildId} = route.params;
```
Comment (Akhror) [inline app/screens/home/DetailsScreen.tsx:66]: Let's add space before headerColor:
const {fromCd, headerColor, fromScreen, backgroundColor,detailData, listCategoryArray, selectedChildActivitiesData, currentSelectedChildId} = route.params;

File: app/components/shared/RelatedArticles.tsx (hunk 2 above)
Comment (Kalyani790) [reply app/components/shared/RelatedArticles.tsx]: done

File: app/components/shared/RelatedVideoArticles.tsx (hunk 1 above)
Comment (Kalyani790) [reply app/components/shared/RelatedVideoArticles.tsx]: done

**Pull request merged**